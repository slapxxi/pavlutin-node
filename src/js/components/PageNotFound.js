import React from 'react';
import Icon from 'react-fontawesome';
import { setTitle } from '../utils';

function PageNotFound() {
  setTitle('Page Not Found');
  return (
    <div className="container page-not-found">
      <h1><Icon name="exclamation-triangle" /> 404 - Page Not Found</h1>
    </div>
  );
}

export default PageNotFound;
