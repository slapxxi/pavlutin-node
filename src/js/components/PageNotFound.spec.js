import React from 'react';
import render from 'react-test-renderer';
import PageNotFound from './PageNotFound';

it('renders Not Found page', () => {
  const tree = render.create(<PageNotFound />);
  expect(tree.toJSON()).toMatchSnapshot();
});
