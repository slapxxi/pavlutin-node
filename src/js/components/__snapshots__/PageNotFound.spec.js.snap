// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Not Found page 1`] = `
<div
  className="container page-not-found"
>
  <h1>
    <span
      aria-hidden={true}
      className="fa fa-exclamation-triangle"
    />
     404 - Page Not Found
  </h1>
</div>
`;
